cmd/nvidia-ctk/cdi/generate/generate.go

```diff
@@ -176,6 +177,12 @@ func (m command) build() *cli.Command {
 			Usage:       "Specify a pattern the CSV mount specifications.",
 			Destination: &opts.csv.ignorePatterns,
 		},
+		&cli.StringSliceFlag{
+			Name:        "skip-hook",
```

Comment:
What about `disable-hook` (we could even add an alias as `disable-hooks`) instead? Thinking ahead, it may be useful to allow users to do something like:
```
nvidia-ctk cdi generate \
    --disable-hook=update-ldcache \
    --disable-hook=enable-cuda-compat \
    --mode=nvml
```

Even further forward, more conservative users may want to consider:
```
nvidia-ctk cdi generate --disable-hooks=all
```
although that is out of scope of this PR.

Reply:
@klueska @jgehrcke  what are your thoughts on the UX?